pkg/diff/chunk.go

```diff
@@ -79,191 +133,335 @@ func getChunksForTable(table *TableInstance, column *model.ColumnInfo, chunkSize
 		return nil, nil
 	}
 
-	chunkCnt := (cnt + int64(chunkSize) - 1) / int64(chunkSize)
+	chunkCnt := (int(cnt) + chunkSize - 1) / chunkSize
```

Comment:
`// use sampling check, can check more fragmented by split to more chunk` means we need to do more queries, what's the benefits?

Reply:
seems no benefits, I will remove this